docs/nlc/alchemy.md

```diff
@@ -0,0 +1,12 @@
+# Alchemy API setup
+
+To have your bot understand and find locations in statements using Alchemy APIs, get started with the following steps:
```

Comment:
Saw the other files, maybe it's better to keep the same structure.

I'm not a grammar wizard, but thought this wasn't a complete sentence, either way, the message is clear to me.